portfolio.html

```diff
@@ -0,0 +1,69 @@
+        </header>
+        <section class="portfolio">
+            <atricle class="main-portfolio">
+                <header><h3>Lorem, ipsum.</h3></header>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus fuga architecto fugit eum totam odit, ex sunt, non dolores, recusandae molestias repudiandae ipsam! Harum, totam, rerum reiciendis officia minima odit ipsum incidunt ut at culpa reprehenderit dolores velit iure in magnam unde explicabo dignissimos architecto error quia amet? Mollitia modi quis cupiditate. Molestias soluta iste aliquam porro aliquid illum distinctio iusto et dolor autem excepturi eum, deleniti ullam nesciunt blanditiis. Sapiente incidunt quo impedit autem suscipit inventore maxime assumenda facere modi odit architecto sunt excepturi illo temporibus asperiores, neque eveniet deleniti debitis eligendi ipsam! Ullam accusantium doloremque necessitatibus nulla iusto!</p>
+                <img width="600px" height="500px" src="images/circuit.avif" alt="circuit board">
+            </atricle>
+            <atricle class="main-portfolio">
+                <header><h3>Lorem, ipsum.</h3></header>
+                <p>Consectetur, cumque. Quod fugiat, sit perspiciatis quae magni quia facere tempore! Reiciendis quod neque hic libero et ullam minus. Quisquam, alias dicta vitae optio fugit repellendus vel. Illum beatae esse tempora id maiores optio perspiciatis, consectetur ex eum sunt pariatur facere sed voluptatem quaerat, consequuntur deleniti porro ea sequi placeat expedita? Quibusdam necessitatibus saepe dignissimos quisquam non. Eveniet unde fugit ipsam iste nihil sit quod, tempora vero, id temporibus nulla rem nam architecto odio autem est fuga consequuntur eaque obcaecati odit aliquid quibusdam animi inventore ipsum. Quas vel vero voluptas, quam odio non iure totam quaerat! Debitis error distinctio explicabo!</p>
+                <img width="600px" height="500px" src="images/synth.avif" alt="synthesizer">
+            </atricle>
+            <atricle class="main-portfolio">
+                <header><h3>Lorem, ipsum.</h3></header>
+                <p>Consectetur, cumque. Quod fugiat, sit perspiciatis quae magni quia facere tempore! Reiciendis quod neque hic libero et ullam minus. Quisquam, alias dicta vitae optio fugit repellendus vel. Illum beatae esse tempora id maiores optio perspiciatis, consectetur ex eum sunt pariatur facere sed voluptatem quaerat, consequuntur deleniti porro ea sequi placeat expedita? Quibusdam necessitatibus saepe dignissimos quisquam non. Eveniet unde fugit ipsam iste nihil sit quod, tempora vero, id temporibus nulla rem nam architecto odio autem est fuga consequuntur eaque obcaecati odit aliquid quibusdam animi inventore ipsum. Quas vel vero voluptas, quam odio non iure totam quaerat! Debitis error distinctio explicabo!</p>
+                <img width="600px" height="500px" src="images/records.jpg" alt="records">
+            </atricle>
```

Comment:
Only noticing small concerns like spelling => article, but again want to stamp the effective use of semantic elements here.